File: fizzbuzz.rb
#!/usr/bin/env ruby

def fizzbuzz(i)
	text = ''
	text << 'fizz' if i % 3 === 0
	text << 'buzz' if i % 5 === 0
	text.empty? ? "#{i}" : text
end

def main()
	for i in 1..100 do
		puts "#{fizzbuzz i}\n"
	end
end

main

File: fizzbuzz.py
#!/usr/bin/env python
# Python 3

def fizzbuzz(i):
	text = ''
	if not i % 3:
		text += 'fizz'
	if not i % 5:
		text += 'buzz'
	return len(text) and text or "%s" % i

def main():
	for i in range(1, 101):
		print(fizzbuzz(i))

main()

